src/FilterWrapper/FilterWrapper.tsx

```diff
@@ -39,76 +53,64 @@ export class FilterWrapper extends React.Component<Props> {
 		);
 	};
 
-	public render() {
-		const {
-			children,
-			statisticsValue,
-			statisticsDescription,
-			applyButtonCaption,
-			clearButtonCaption,
-			doesContainFilter,
-			onApply,
-			onClear,
-			selectionState,
-			isDataOutdated,
-			filterActions,
-			filterActionsCaption,
-			scrollState
-		} = this.props;
-
-		return (
-			<div
-				className={cn("kit-filter", {
-					"kit-filter_short": !doesContainFilter,
-					"kit-filter_with-filter-action": filterActions.length
-				})}
-			>
-				<div className="kit-filter__top-filter">
-					<FilterActionsPopover
-						filterActionsCaption={filterActionsCaption}
-						filterActions={filterActions}
-					/>
-				</div>
-				<ul className="kit-filter__all-wrap">{children}</ul>
-				{doesContainFilter ? (
-					<div className="kit-filter__wrap">
-						<div className="kit-filter__wrap-filter">
-							{scrollState !== "minfied" && (
+	return (
+		<>
+			<div ref={refBreackPoint} className="kit-filter__breackpoint" />
+			<FilterWrapperContext.Provider value={refBreackPoint.current}>
```

Comment:
Думал везде поправишь 😕
```suggestion
			<FilterWrapperContext.Provider value={refBreakPoint.current}>
```